src/index.css

```
@@ -86,3 +86,90 @@ blockquote {
 .components-popover__content {
 	width: 100%;
 }
+
+#assistant-markdown {
+	line-height: 1.6;
+	word-wrap: break-word;
+}
+
+#assistant-markdown blockquote {
+	font-weight: bold;
+	margin: 1rem 0;
+	padding: 0.5rem 1rem;
+}
+
+#assistant-markdown code {
+	border-radius: 3px;
+	margin: 1rem 0 !important;
+	white-space: pre-wrap;
```

Let's add font-size: 13px to keep the same size as the design.

```suggestion
	white-space: pre-wrap;
    font-size: 13px;
```